_includes/_wallet_install.htm

```diff
@@ -6,30 +6,15 @@
                 <h4 class="text-bold">
                     Install the Gridcoin client
                 </h4>
-                <h4>Windows & Linux</h4>
+                <h4>Wallet Download</h4>
                 <a id="64-bit-windows" class="btn rounded-pill grcbtn m-1" style="display:none"> 
-                    64-bit Windows
+                    Download Gridcoin wallet
                 </a>
-                <a id="32-bit-windows" class="btn rounded-pill grcbtn m-1" style="display:none">
-                    32-bit Windows
-                </a>
-                <!-- buttons above work by the fetch-release.js script -->
-                <br>
-                <a class="btn rounded-pill grcbtn m-1" href="https://github.com/gridcoin-community/Gridcoin-Research/releases/latest">
-                    Download via GitHub
+                <a class="btn rounded-pill grcbtn m-1" href="/#Downloads">
+                    Download Gridcoin wallet
                 </a><br />
-
-                <h4>MacOS</h4>
-                <a class="btn rounded-pill grcbtn m-1" href="https://github.com/Git-Jiro/homebrew-jiro/releases">
```

Comment:
why is this being removed?

Reply:
It's just linking all platforms to the Gridcoin download page. Previously, it linked to the Github releases page, which is a little more difficult to navigate.

Reply:
Git-Jiro's release page is different from the main one

Reply:
I'm not sure I understand what you're saying, can you clarify please? It seems to me the gridcoin.us download page ought to be a good authoritative place to point users for downloads, no?

Reply:
Ah nevermind, I thought this was redirecting to GitHub

Reply:
The opposite in fact, it used to sent people to github releases, these edits make it send people to the gridcoin.us download page instead :)